tests/cli/test_checkpoint.py

```diff
@@ -1263,7 +1263,10 @@ def test_checkpoint_run_on_non_existent_validations(
     assert result.exit_code == 1
 
     stdout: str = result.stdout
-    assert 'Checkpoint "no_validations" does not contain any validations.' in stdout
+    assert (
+        'Checkpoint "no_validations" must contain either a batch_request or validations.'
+        in stdout
+    )
```

Comment:
Really like the change with the messaging here.. helps the user better-understand whether this is a good-or-bad result